src/protagonist/DLCS.AWS/SNS/TopicPublisher.cs

```diff
@@ -98,17 +99,29 @@ public class TopicPublisher : ITopicPublisher
             return false;
         }
     }
-
-    private static Dictionary<string, MessageAttributeValue> GetMessageAttributes(ChangeType changeType)
+    
+    private static Dictionary<string, MessageAttributeValue> GetMessageAttributes(ChangeType changeType, bool engineNotified)
     {
         var attributeValue = new MessageAttributeValue
         {
             StringValue = changeType.ToString(),
             DataType = "String"
         };
-        return new Dictionary<string, MessageAttributeValue>
+        var messageAttributes = new Dictionary<string, MessageAttributeValue>
         {
             { "messageType", attributeValue }
         };
+        
+        if (engineNotified)
+        {
+            messageAttributes.Add(new KeyValuePair<string, MessageAttributeValue>("EngineNotified",
```

Comment:
Should this be `"engineNotified"`? Other used attribute is `"messageType"` so having camelCase for some and PascalCase for others could lead to possible issues reading them as they're case sensitive.